game/yahtzee.h

```diff
@@ -0,0 +1,24 @@
+#include <iostream>
+#include "die.h"
+
+class Yahtzee {
+public:
+    Yahtzee();
+    ~Yahtzee() {}
+    void roll(int diceToRoll[5]);
+    void scoreRound(int scoringClass);
+    void startGame();
+private:
+    Die gameDie = Die(6);
+    int dice[5];
+    int usedClasses[13] = {0};
+    int score = 0;
```

Comment:
underscores

Reply:
This means I should use underscore naming for private members? ie "_score" instead of "score"?